src/__tests__/locators/utils/__mocks__/locatorsList.mock.ts

```diff
@@ -1,16 +1,19 @@
 import { LocatorTaskStatus } from "../../../../features/locators/types/locator.types";
 import { MUIclasses } from "../../../../features/locators/types/muiClasses.types";
 import { LocatorValidationErrorType } from "../../../../features/locators/types/locator.types";
+import { LocatorType } from "../../../../common/types/common";
 
 export const locatorsListMock = [
   {
     element_id: "7967190244322359771369749968_0",
     generate: true,
     pageObj: 0,
+    locatorType: LocatorType.xPath,
     parent_id: "",
     jdnHash: "7967190244322359771369749968",
     locator: {
-      fullXpath: "/html/body/header/div/nav/ul[1]",
+      cssSelector: null,
```

Comment:
I didn't see a point to generate css selectors within xPaths and other way round, because I think it's extra and user may never use selector or xPath that we generated and kept for they. 
May be I miss smth.

Reply:
the css selectors are generating under the hood with the xpath generation, by default they can't be empty. for custom locators will be added in #1140

Reply:
Yes, I meant that we may don't generate css selectors within xPath generation, but do it when user will change locatorType

Reply:
or, probably, you've already done it in `getLocatorValueOnTypeSwitch`